### DISCUSSION TITLE
Dịch eager mode, eager execution

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#138] [Terminology] [answered]
Nên dịch từ `eager` là `sẵn sàng` hay `ngay` nhỉ?
`sẵn sàng` thì xuôi tai cho cả hai từ trên:
* `trạng thái sẵn sàng`
* `sẵn sàng thực thi`

**Comment by rootonchair** [comment]
2 từ này khó quá giữ nguyên đc ko nhỉ :(

**Comment by minhduc0711** [comment, marked as answer]
Dịch "eager" là "tức thời" thì sao nhỉ? Ám chỉ việc operation trả kết quả tức thời luôn.

**Reply by quangvnai** [reply]
Thế mà lại k nghĩ tới từ `tức thời` này, từ `ngay` thì nghe thô quá :)) Vậy chốt từ này nhé ;)
